### DISCUSSION TITLE
How to filter on the size of a scalar list?

### DISCUSSION THREAD

**Discussion by boredland** [prisma/prisma#22974] [Q&A]
### Question

I have a model that has a Scalar field:

```
  matchIds  String[] @db.ObjectId
```

Is it possible to filter this list by length (e.g. all entities with at least 3 `matchIds`)? In mongodb, that is [easily](https://www.mongodb.com/docs/manual/reference/operator/query/size/) possible, in pg afaik too via `JSONB_ARRAY_LENGTH`. For prisma I seem to be only able to filter on emptyness via [`isEmpty`](https://www.prisma.io/docs/orm/reference/prisma-client-reference#scalar-list-filters)?

**Comment by ludralph** [comment]
Hi @boredland 👋 

Thank you for raising this question.

At the moment this feature is not currently not available. I did find an [issue](https://github.com/prisma/prisma/issues/8935) that is similar to what you are trying to do. Can you please add your usecase and leave a 👍  to help our engineers prioritize it.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by boredland** [reply]
While the use-case mentioned is important too, I think this one is much simpler. This isn't about another relation to be counted, but about a simple array-list.

**Reply by ludralph** [reply]
Hi @boredland, this is currently not possible. Can you please open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for it so our engineers can take a look?

**Reply by boredland** [reply]
-> https://github.com/prisma/prisma/issues/23016

Even though I have tons of issues subscribed that are kinda vital to me and have been open for ages, like the one you referenced.
